moveit_experimental/jog_arm/README.md

```diff
@@ -0,0 +1,15 @@
+Usage:
```

Comment:
Can you expand this README significantly? Open source software is only as good as its documentation

- Add a title using syntax ``# Job Arm``
- Link to the videos in your PR
- Discuss more about what parameters might need to be adjusted
- Link to the spacenav device you recommend to be used.